lib/grantinee/engine/abstract_engine.rb

```diff
@@ -3,7 +3,7 @@
 module Grantinee
   module Engine
     class AbstractEngine
-      NOT_IMPLEMENTED = "Not implemented".freeze
+      NOT_IMPLEMENTED = "Not implemented"
```

Comment:
Why unfreeze? I think rubocop complained about that earlier :D

Reply:
Rubocop complained about redundant freeze: https://circleci.com/gh/blinkist/grantinee/146?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
Aaaah yeah :-) #brainfart